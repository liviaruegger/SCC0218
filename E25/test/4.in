100
4 557
941 153
536 679
486 924
676 640
543 618
992 625
469 371
316 192
605 414
664 909
794 198
29 948
441 407
136 961
437 94
218 916
840 963
223 660
570 164
188 813
491 831
986 48
95 388
646 124
183 500
770 293
228 507
688 338
308 870
549 328
310 274
666 899
654 555
604 856
646 290
634 750
551 394
368 296
140 375
636 907
268 25
593 734
372 41
140 765
679 734
45 821
107 115
38 846
657 292
348 298
901 877
439 702
9 836
813 441
465 624
91 811
258 986
38 670
769 135
230 584
679 195
482 800
701 266
393 942
831 889
550 99
380 157
172 860
518 233
224 517
296 742
668 81
593 497
477 815
371 183
881 884
840 149
2 100
919 467
322 465
924 305
345 479
563 465
306 674
157 821
767 555
741 665
222 737
745 955
640 361
501 24
694 608
326 926
195 111
509 790
980 152
173 119
810 351
648 974
