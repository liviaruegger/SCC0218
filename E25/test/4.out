3609.814056
